### ISSUE TITLE
Namespace appended twice with Hiera plugin

### ISSUE THREAD

**Issue by agarstang** [crayfishx/jerakia#60]
Namespace appended twice with Hiera plugin.

Jerakia Version `1.1.1`

From Jerakia.log:
```
[hiera] backend invoked for key sql_connections using namespace ["profile", "test"]
...
Datasource provided {"blah"=>"blah"} looking for key profile::test::profile::test::sql_connections 
```

Lookup:
```Ruby
lookup :hiera, :use => [ :hiera ] do
    datasource :file, {
      :format => :yaml,
      :docroot => "/etc/puppetlabs/code/environments/#{scope[:environment]}/hieradata",
      :searchpath => [
        "nodes/#{scope[:trusted]['extensions']['pp_datacenter']}/#{scope[:trusted]['certname']}",
        "roles/#{scope[:trusted]['extensions']['pp_role']}",
        "regions/#{scope[:trusted]['extensions']['pp_region']}/common",
        "accounts/#{scope[:trusted]['extensions']['pp_datacenter']}/common",
        "global/os/#{scope[:osfamily]}",
        "global/common"
      ]
    }
```

**Comment by agarstang** [comment]
Confirmed with @crayfishx in private that this is a bug.

Issue seems to be with the source `request` object that lookups clone to their local object to ensure they get their own `request` object and do not tamper with other lookups.

**Comment by agarstang** [comment]
Resolved in [1.1.2](https://github.com/crayfishx/jerakia/milestone/18).
